fenix/app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -211,34 +218,86 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
             owner = viewLifecycleOwner,
             view = binding.root,
         )
+
+        setTranslationFragmentResultListener()
+    }
+
+    private fun setTranslationFragmentResultListener() {
+        setFragmentResultListener(
+            TRANSLATION_KEY,
+        ) { _, result ->
+            result.getString(TRANSLATION_STATUS_STATE_KEY)?.let { translationStatus ->
+                result.getString(SESSION_ID)?.let {
+                    if (it == getCurrentTab()?.id) {
+                        if (translationStatus == IN_PROGRESS) {
+                            FenixSnackbar.make(
+                                view = binding.dynamicSnackbarContainer,
+                                duration = Snackbar.LENGTH_SHORT,
+                                isDisplayedWithBrowserToolbar = true,
+                            )
+                                .setText(requireContext().getString(R.string.translation_in_progress_snackbar))
+                                .show()
+                        }
+
+                        val updateContentDescriptionSelected = result.getString(
```

Comment:
Moving the `State` added in this patch into the `BrowserStore` should allow us to access the state from the Store instead of needing to communicate it between fragments using a bundle.

Reply:
This will not work. The request from figma is https://www.figma.com/file/GT6oZLFw32LiCwKl5GOMWD/Full-Page-Translation?type=design&node-id=7231-206075&mode=design&t=MP4I9hWYk1TLaTp0-0
"Translating snackbar - This will appear if the bottom sheet is dismissed during a translation that hasn’t completed yet." 
I need to know somehow that the user closes the bottom sheet when translation is in progress.